cloudrun-malware-scanner/server.js

```diff
@@ -131,21 +109,17 @@ async function handleGcsObject(req, res) {
           file.bucket);
       return;
     }
-    const config =
-        BUCKET_CONFIG.buckets.filter((c) => c.unscanned === file.bucket)[0];
-    if (!config) {
-      handleErrorResponse(res, 200,
-          `Bucket name - ${file.bucket} not in config`);
-      return;
-    }
 
     const gcsFile = storage.bucket(file.bucket).file(file.name);
     // File.exists() returns a FileExistsResponse, which is a list with a
     // single value.
     if (! (await gcsFile.exists())[0]) {
       // Warn but return successful to client.
       logger.warn(`Ignoring no longer existing file: gs://${file.bucket}/${file.name}`);
-      res.json({status: 'deleted'});
+      handleErrorResponse(
```

Comment:
can u fix line 96 and line 100 and line 105 to return actual errors when request is invalid?
something like 400 error code?